solana-kotlin/src/commonMain/kotlin/net/avianlabs/solana/domain/core/AccountKeysList.kt

```diff
@@ -1,6 +1,10 @@
 package net.avianlabs.solana.domain.core
 
-internal fun List<AccountMeta>.normalize(): List<AccountMeta> = groupBy { it.publicKey }
+import net.avianlabs.solana.tweetnacl.ed25519.PublicKey
+
+internal fun List<AccountMeta>.normalize(
```

Comment:
would it make sense to have two separate functions ? 
`normalize()` and `normalizeWithFeePayer` ?

Reply:
no, fee payer might be set or not